app/presenters/digest_presenter.rb

```diff
@@ -0,0 +1,75 @@
+class DigestPresenter < NotificationPresenter
+  include AvatarHelper
+
+  attr_accessor :notifications, :template
+
+  def initialize(notifications, template)
+    @notifications = notifications
+    @template = template
+  end
+
+  def comment_path(anchor: false)
+    polymorphic_url(
+      path_to_comment,
+      anchor: anchor
+    )
+  end
+
+  def created_at_ago
+    # We can't use the local_time gem here because there's no JS
+    "#{time_ago_in_words(notification.created_at)} ago"
+  end
+
+  private
+
+  def avatar_image(size)
+    if notification.actor
+      h.image_tag(
+        avatar_url(notification.actor, size: size),
+        alt: notification.actor.email,
+        class: 'gravatar',
+        title: notification.actor.email,
+        width: size,
+        # HACK: we can't use data-fallback-image for setting the fallback image
+        # because there's no JS in the mail. Instead, we're relying on onerror
+        # callback to re-set the image tag's src attribute.
+        onerror: "this.src = '#{image_path('logo_small')}';"
```

Comment:
Shouldn't we default to `profile`?

```suggestion
        onerror: "this.src = '#{image_path('profile')}';"
```

However see my note in the mailer re: using proper file names for the attachments.